src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -352,7 +352,7 @@ namespace winrt::TerminalApp::implementation
                     {
                         auto controlSettings = activeControl.Settings().as<TerminalSettings>();
                         controlSettings->ApplyColorScheme(scheme);
-                        activeControl.UpdateSettings(*controlSettings);
+                        activeControl.UpdateSettings();

```

Comment:
So just to get this straight - if the user does something like:
* set the color table with `colortool -x whatever`
* Then change the scheme with the `SetColorScheme` action

Then nothing will actually change for the `SetColorScheme` action, because the palette was already overridden by the control at runtime?

(not saying this is wrong, just want to get our ducks in a row)

Reply:
`SetColorScheme` at runtime will _also_ affect the runtime terminal settings, so it should override the previous override (`SetColorScheme` acts on the TS object that lives in the control)

Reply:
Oh that's a good point! That's good to know. Thanks!